webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/Info/IUserFormInfoState.ts

```diff
@@ -1,17 +1,20 @@
 /*
  * CloudBeaver - Cloud Database Manager
- * Copyright (C) 2020-2024 DBeaver Corp and others
+ * Copyright (C) 2020-2025 DBeaver Corp and others
  *
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
 
-export interface IUserFormInfoState {
-  userId: string;
-  enabled: boolean;
-  password: string;
-  metaParameters: Record<string, any>;
-  teams: string[];
+import { schema } from '@cloudbeaver/core-utils';
 
-  authRole: string; // used in TE product
-}
+export const USER_FORM_INFO_PART_SCHEMA = schema.object({
+  userId: schema.string().trim(),
+  enabled: schema.boolean(),
+  password: schema.string().trim(),
+  metaParameters: schema.record(schema.string(), schema.string().trim().or(schema.any())),
+  teams: schema.array(schema.string().trim()),
+  authRole: schema.string().trim(),
```

Comment:
Probably, we don't need to trim these fields because the user does not manually specify them (we need to trim only fields that the user manually types)